src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/endpoint/UserEndpoint.java

```diff
@@ -189,6 +190,18 @@ public UserEndpoint(
     return ResponseEntity.ok(createdUser);
   }
 
+  @DeleteMapping("/{userIdentity}")
+  public ResponseEntity<UserDTO> deleteUser(
```

Comment:
if you want to ensure the delete and the audit get committed together, you will need to make the method transactional `@Transactional` . I think there is a general problem here with these endpoint methods calling multiple database operations outside a single transaction, but you don't need to solve the other stuff here.

Reply:
done